pkgs/servers/sql/cockroachdb/default.nix

```
@@ -1,70 +1,24 @@
-{ lib, stdenv, buildGoPackage, fetchurl
-, cmake, xz, which, autoconf
-, ncurses6, libedit, libunwind
-, installShellFiles
-, removeReferencesTo, go
-}:
-
-let
-  darwinDeps = [ libunwind libedit ];
-  linuxDeps  = [ ncurses6 ];
-
-  buildInputs = if stdenv.isDarwin then darwinDeps else linuxDeps;
-  nativeBuildInputs = [ installShellFiles cmake xz which autoconf ];
-
-in
-buildGoPackage rec {
-  pname = "cockroach";
-  version = "20.1.8";
-
-  goPackagePath = "github.com/cockroachdb/cockroach";
-
-  src = fetchurl {
-    url = "https://binaries.cockroachdb.com/cockroach-v${version}.src.tgz";
-    sha256 = "0mm3hfr778c7djza8gr1clwa8wca4d3ldh9hlg80avw4x664y5zi";
+# https://www.cockroachlabs.com/docs/releases/release-support-policy.html
+
+self: {
```

This I find strange. Why not `{ callPackage }: {`?

---

I honestly just copy / pasted from the other PR. I'm sure things can be cleaned up / done differently.

---

I had a look around, and I think it should be `{ callPackage }: {` here, and then `callPackage` instead of `import` in `all-packages.nix` as well.

```suggestion
{ callPackage }: {
```